login-fire-social.html

```diff
@@ -158,7 +161,7 @@
         type: Array,
         readOnly: true,
         computed: '_computeProviders(providers)'
-      }
+      },
```

Comment:
It's not really intentional. Sometimes it happens because I may add a property or function after while testing and I miss it after deletion but sometimes I add it thinking that I wouldn't have to remember to add a comma if I add another property/function after.
Thanks for spotting them, I will remove them to be more consistent everywhere.